bindata/bootkube/manifests/00_etcd-endpoints-cm.yaml

```diff
@@ -2,6 +2,10 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: etcd-endpoints
-  namespace: openshift-etcd
+  namespace: openshift-etcd {{ if not .BootstrapScalingStrategy.InPlace }}
   annotations:
     alpha.installer.openshift.io/etcd-bootstrap: {{ .BootstrapIP }}
+{{ end }} {{ if .BootstrapScalingStrategy.InPlace }}
```

Comment:
(No action required) Maybe start the new substitution block on the next line?

Reply:
it seemed to inject a newline but I want to play with that so its cleaner I agree.

Reply:
fixed